client/js/components/GameView/Card.vue

```diff
@@ -204,6 +204,9 @@ export default {
   right: 0;
   z-index: 1;
 }
```

Comment:
Added border-radius to match other radius on cards. 
![image](https://user-images.githubusercontent.com/47369872/206083442-0e86be92-e8d3-40e7-a263-f278794a6919.png)
![image](https://user-images.githubusercontent.com/47369872/206083471-1de00505-af2a-4be2-9e20-562f57d9a4f2.png)